services/provider/server/server.go

```diff
@@ -422,20 +422,27 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 	return extR, nil
 }
 
-func (s *OCSProviderServer) getCephClientSecretName(ctx context.Context, name string) (string, error) {
+func (s *OCSProviderServer) getCephClientInformation(ctx context.Context, name string) (string, string, string, error) {
 	cephClient := &rookCephv1.CephClient{}
 	err := s.client.Get(ctx, types.NamespacedName{Name: name, Namespace: s.namespace}, cephClient)
 	if err != nil {
-		return "", fmt.Errorf("failed to get rook ceph client %s secret. %v", name, err)
+		return "", "", "", fmt.Errorf("failed to get rook ceph client %s secret. %v", name, err)
 	}
 	if cephClient.Status == nil {
-		return "", fmt.Errorf("rook ceph client %s status is nil", name)
+		return "", "", "", fmt.Errorf("rook ceph client %s status is nil", name)
 	}
 	if cephClient.Status.Info == nil {
-		return "", fmt.Errorf("rook ceph client %s Status.Info is empty", name)
+		return "", "", "", fmt.Errorf("rook ceph client %s Status.Info is empty", name)
 	}
 
-	return cephClient.Status.Info["secretName"], nil
+	if len(cephClient.Annotations) == 0 {
+		return "", "", "", fmt.Errorf("rook ceph client %s annotation is empty", name)
+	}
+	if cephClient.Annotations["ocs.openshift.io.storageclaim"] == "" || cephClient.Annotations["ocs.openshift.io.cephusertype"] == "" {
```

Comment:
Use the existing constants instead of hard coded strings 
```suggestion
	if cephClient.Annotations[storageClaimAnnotation] == "" || cephClient.Annotations[storageCephUserTypeAnnotation] == "" {
```